### PR TITLE
Adds Uncle Pete's Rollerdome, a replacement for Disco Inferno

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

There were a few folks who mentioned that they wanted to keep Disco Inferno for the dance stuff, not the fire hell stuff. I made it themed off of a 90s roller rink for giggles. Here's a version of Disco Inferno without the inferno.

image

## Why It's Good For The Game

It's fun and people seem to like it.

## Changelog
:cl:
add: A new dance-themed emergency shuttle has been added to the rotation. Buy it today!
/:cl:

### CODE REVIEW

**Comment by Winterous** [comment]
I hope one of the arcades is guaranteed to be medbot's amputation adventure. If not, it should be.

**Comment by ExcessiveUseOfCobblestone** [comment]
are there heelies so you can crash into people as  you go down the single file hellhole?

**Comment by TheVekter** [comment]
The vending machines have the reskinned wheelies in them

Also I might w i d e n this a bit

**Review by Mothblocks — DISMISSED** [review]

**Comment by Farquaar** [comment]
This is great.

**Comment by wesoda25** [comment]
I feel as if you should use some of those nice tile sidings to edge in the skating rink

**Comment by gitbirb** [comment]
thank god for this. Now i won't be called a griefer because i like the disco ball and as a result love disco inferno. To add some spice though, why not add a few goats renamed to "Dancing goats" or something on the dancing floor, or maybe expand the dancing floor a bit since i don't think a whole crew will be able to sit on it

**Comment by TheVekter** [comment]
> I feel as if you should use some of those nice tile sidings to edge in the skating rink (and the bar) to help the transition

They're there - you can see them in the map diff. Not sure why they aren't showing in the screenshot. I'll mess with it later.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -372,6 +372,14 @@
 	description = "The Nanotrasen Emergency Shuttle Port(NES Port for short) is a shuttle used at other less known Nanotrasen facilities and has a more open inside for larger crowds, but fewer onboard shuttle facilities."
 	credit_cost = 500
 
+/datum/map_template/shuttle/emergency/rollerdome
+	suffix = "rollerdome"
+	name = "Uncle Pete's Rollerdome"
+	description = "Developed by a member of Nanotrasen's R&D crew that claims to have travelled from the year 2028. \
+	He says this shuttle is based off an old entertainment complex from the 1990s, though our database has no records on anything pertaining to that decade."
```
Comment (ATH1909) [inline code/datums/shuttles.dm:379]: uh, from a lore perspective, are you SURE that nanotrasen has no records on anything from the 90s? characters will often try to play off accidental/intentional references to real world events by saying they happened hundreds of years ago/during an era they're studying as a hobby, but if nanotrasen has no records whatsoever on an entire decade that's close to many modern day references, that could be a problem...

Comment (TheVekter) [reply code/datums/shuttles.dm:379]: I guess I can change it but I wasn't aware we were taking everything in the shuttle category as canon.

**Pull request merged**